tests/console/yast2_cmdline.pm

```diff
@@ -50,7 +50,7 @@ sub run {
 
     # Run YaST CLI tests
     run_yast_cli_test('yast2-network');
-    run_yast_cli_test('yast2-dns-server');
+    run_yast_cli_test('yast2-http-server');
```

Comment:
That does not look like related to the subject line of the PR

Reply:
yast2-dns-server ruby test case assumes that bind package as pre-installed. This test won't run successfully with previous setup. Therefore I have change the module.
http://dhcp151.suse.cz/tests/3369#step/yast2_cmdline/32

Reply:
There should be no issue to install bind, but in general it's fine to use other modules. Maybe we need a bug that bind is not in dependencies of yast2-dns-server. And I guess @okurz means that we need another subject line for PR, e.g. "Fix yast2_cmdline test module"

Reply:
I have tried to run yast2 dns-server without bind and got this message.
``These packages need to be installed: bind``
I am not really sure whether rpm dependency is necessary in this case. It looks like a redundant solution to me. We should also bear in mind that we are executing their own ruby test suits with this module, which are running fine in OBS.

Reply:
Exactly, that's why I've said that using http still does the trick as we mainly want to verify that users will be able to run these tests

Reply:
Huh, ok. I saw the purpose of the test module differently but works for me. Please update the summary line of the test module accordingly then